docs/monitoring/monitoring-troubleshooting.asciidoc

```diff
@@ -22,3 +22,28 @@ The following error appears in a banner at the top of the screen in {kib} on the
 You cannot monitor a version 6.3 or later cluster from {kib} version 6.2 or earlier. 
 To resolve this issue, upgrade {kib} to 6.3 or later. See 
 {stack-ref}/upgrading-elastic-stack.html[Upgrading the {stack}]. 
+
+[float]
+=== No monitoring data is visible in {kib}
+
+*Symptoms:*
+
+The *Monitoring* page in {kib} is empty. 
+
+*Resolution:*
+
+. Confirm that {kib} is seeking monitoring data from the appropriate URL. By 
+default, data is retrieved from the cluster specified in the `elasticsearch.url`
+setting in the `kibana.yml` file. If you want to retrieve it from a different 
+monitoring cluster, set `xpack.monitoring.elasticsearch.url`. See
+<<monitoring-settings-kb>>.
+
+. Confirm that there is monitoring data available at that URL. Depending on 
+which method you use to collect it and which product version you're using, it is
+stored in either `.monitoring-*` or `metricbeat-*` indices. At a minimum, you
```

Comment:
Actually no monitoring data is being stored in `metricbeat-*` indices yet. And it's unlikely to be the case for 7.0 as well. So I think we shouldn't mention those indices here just yet.

Reply:
Thanks for the feedback!  I've changed it to this: "It is stored in
indices such as `.monitoring-kibana-*` and `.monitoring-es-*`. "